launch-script/src/proposals/deploy_evm_loader_from_buffer.rs

```diff
@@ -0,0 +1,44 @@
+// =========================================================================
+// Deploy evm_loader from buffer account
+// =========================================================================
+use solana_sdk::{
+    account::Account,
+    bpf_loader_upgradeable,
+};
+use spl_governance::state::proposal_transaction::InstructionData;
+use crate::prelude::*;
+
+pub fn deploy_evm_loader_from_buffer(_wallet: &Wallet, client: &Client,
+        transaction_inserter: &mut ProposalTransactionInserter,
+        cfg: &Configuration,
+        evm_loader_pubkey: Pubkey,
+        buffer_pubkey: Pubkey,
+        upgrade_authority_pubkey: Pubkey,
+) -> Result<(), ScriptError> {
+
+    let buffer_account_opt: Option<Account> = client.get_account(&buffer_pubkey)?;
+    if let Some(buffer_account) = buffer_account_opt {
+        let data_len: usize = buffer_account.data.len();
+        let minimum_balance_for_rent_exemption = cfg.client.get_minimum_balance_for_rent_exemption(data_len).unwrap();
+
+        let instructions: Vec<InstructionData> =
+            bpf_loader_upgradeable::deploy_with_max_program_len(
+                &client.payer.pubkey(),
```

Comment:
We have a problem here. Because the proposal should be executed by any person, we can't require to sign instructions inside the proposal by anyone except governance. But the governance account can't fund new accounts. For TokenGenesisEvent this problem was resolved by pre-fund all accounts by the payer (before creating the proposal).
The `deplot_with_max_program_len` instruction do not let this way (because it re-charge (!!!) account).
Moreover, we need to sign the transaction with the `evm_loader` private key.

It seems the better way will be to deploy the NeonEVM program with some empty image and pass it under maintenance governance. And after this governance can check all environments and decide to run NeonEVM. This way no one can block proposal execution after approval.

Reply:
Rewritten using bpf_loader_upgradeable::upgrade command.